terraform-infra-approvals/global.json

```diff
@@ -8,7 +8,9 @@
     {"type": "azurerm_user_assigned_identity"},
     {"type": "random_string"},
     {"type": "azurerm_subnet"},
-    {"type": "azurerm_api_management"}
+    {"type": "azurerm_api_management"},
+    {"type": "azurerm_management_lock"},
```

Comment:
Is this a good idea won’t terraform remove this if a module gets removed accidentally?